CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,20 @@
 
 import Foundation
 
-print("Hello, World!")
+func ice() {
+    for _ in 1...8{
+        print("***********")
+    }
+}
+
+func bar(){
+    for _ in 1...4{
+        print("    | |")
+    }
```

Comment:
전체적으로 반복문을 잘 사용하신 것 같습니다 👍👍👍

일반적으로 아래와 같이 중괄호는 띄워서 작성하는 것 같아요!!
```suggestion
    for _ in 1...4 {
        print("    | |")
    }
```